code/SharedFunctionality.UI/Services/ProjectMetadataService.cs

```diff
@@ -38,12 +38,12 @@ public static ProjectMetadata GetProjectMetadata(string projectPath)
             {
                 if (!string.IsNullOrEmpty(projectPath))
                 {
-                    var metadataFileNames = new List<string>() { "Package.appxmanifest", "WTS.ProjectConfig.xml" };
+                    var metadataFileNames = new List<string>() { "Package.appxmanifest", "TemplateStudio.xml" };
                     var metadataFile = metadataFileNames.FirstOrDefault(fileName => File.Exists(Path.Combine(projectPath, fileName)));
                     if (!string.IsNullOrEmpty(metadataFile))
                     {
                         var manifest = XElement.Load(Path.Combine(projectPath, metadataFile));
-                        XNamespace ns = "http://schemas.microsoft.com/appx/developer/windowsTemplateStudio";
+                        XNamespace ns = "http://schemas.microsoft.com/appx/developer/templatestudio";
```

Comment:
It seems odd that this line is even in the file as it's a duplicate from line 31 (ignoring case)

Reply:
Going to merge as is for now.